Source/ee/EeExecutor.cpp

```diff
@@ -60,7 +60,7 @@ void CEeExecutor::AddExceptionHandler()
 	sigemptyset(&sigAction.sa_mask);
 	int result = sigaction(SIGSEGV, &sigAction, nullptr);
 	assert(result >= 0);
-#elif defined(__APPLE__)
+#elif defined(__APPLE__) && !TARGET_OS_TV
```

Comment:
Is there really no way to use the `mach_*` functions on tvOS? Without the exception handler, it's probably gonna cause unexpected issues due to JIT code mismatch.

Reply:
Let me do some more research and get back to you. Those calls have deliberately been made unavailable on tvOS but let me see if there are alternatives.

Reply:
Both mach_msg and mach_msg_override are unavailable to tvOS. It seems that ppsspp just uses the standard posix signal() handling, which seems to work here as well. I've updated the PR. Thank you for asking me to take another look!

Reply:
Humm, I've tried the `signal` based exception handling and I'm not quite sure it works on Apple platforms. I've tried it on macOS and the signal doesn't get processed correctly and the emulator will just shut down. You can try FF10, I think that one will cause some exceptions to be triggered and the emulator will crash.

I think you're probably better off defining `DISABLE_PROTECTION` or going back to your previous version which removed the `mach_*` calls (also make sure to remove `mprotect`).

Reply:
Thanks, let me experiment more. tvOS is quite a bit different from macOS (and slightly different from iOS in subtle ways, though it is considered an iOS variant).